accountCredentials.md

```diff
@@ -17,6 +10,40 @@ Specific information for finding your credentials can be found [here](https://su
+| `userId`                               | Your Bandwidth Account ID. The `userId` is used as part of the url to make API requests. This may be the same value as `accountId`. <br> EX: `https://messaging.bandwidth.com/api/v2/users/{userId}/messages` <br><br> ⚠️ This **IS NOT** the same as your log in! ⚠️                                                                                |  `u-sd89g93epa2nb4euazd83kv8` OR `920012` |
 | `apiToken`                             | Your unique api token.  The `apiToken` is used as the **username** to authenticate to the API as part of the basic auth scheme                                                                                                                                                                       |  `f12a9edeed04ecd21b303c6f1f9f0831a1482f7f3c59199e`          |
 | `apiSecret`                            | Your unique api secret.  The `apiSecret` is used as the **password** to authenticate to the API as part of the basic auth scheme                                                                                                                                                                     |  `j54935lddasl837592356aasdf8359hlo3` |
+| `applicationId` | The ID of your messaging application | `532qd-fk5odk5-dlslka40-l5k3lsdmc` |
+
+## Authentication on each API
+
+### Voice
+
+| Credential Name  | How it is used |
+|:---------------------------------------|:-------------------|
+| `accountId`                             | Used as a URL parameter for the API endpoint `https://voice.bandwidth.com/v2/account/{accountId}`   |
+| `apiToken`                             | Used as the username for HTTP basic auth  | 
+| `apiSecret`                             | Used as the password for HTTP basic auth  | 
+
+### Messaging
+
+| Credential Name  | How it is used |
+|:---------------------------------------|:-------------------|
+| `userId`                             | Used as a URL parameter for the API endpoint `https://messaging.bandwidth.com/api/v2/{userId}/messages`   |
+| `apiToken`                             | Used as the username for HTTP basic auth  | 
+| `apiSecret`                             | Used as the password for HTTP basic auth  | 
+| `applicationId`        | Used as a parameter in the request body  | 
+
+### Numbers
+
+| Credential Name  | How it is used |
+|:---------------------------------------|:-------------------|
+| `accountId`  | Used as a URL parameter for the API endpoint `https://dashboard.bandwidth.com/api/accounts/{accountId}`   |
+| `username` | Used as the username for HTTP basic auth  | 
+| `password`  | Used as the password for HTTP basic auth  | 
+
+## Guides for finding and creating credentials
+* [Token and Secret](https://support.bandwidth.com/hc/en-us/articles/360014110974)
+* [Application ID](https://support.bandwidth.com/hc/en-us/articles/360013007654)
+
+<br><br>
```

Comment:
needed?

Reply:
Without these breaks the bottom of the page looks like this. People have commented in the past that it looks like information is cut off

![Screen Shot 2019-06-19 at 9 25 20 AM](https://user-images.githubusercontent.com/39915377/59769445-2bc98700-9274-11e9-8106-98765ba978f6.png)

Reply:
Maybe 2 `<br>` tags was too much. I changed it to 1. This is how it looks
![Screen Shot 2019-06-19 at 9 27 23 AM](https://user-images.githubusercontent.com/39915377/59769603-7945f400-9274-11e9-9d9c-1ec1f62195a6.png)